kernel/arch/dreamcast/include/arch/timer.h

```diff
@@ -15,15 +16,17 @@
     the gettime functions.
 
     \author Megan Potter
+    \author Falco Girgis
 */
 
 #ifndef __ARCH_TIMER_H
 #define __ARCH_TIMER_H
 
+
+#include <stdint.h>
```

Comment:
Ive always seen the includes inside the _BEGIN_DECLS or whatever.  There a difference?